### ISSUE TITLE
Price per person instead of room

### ISSUE THREAD

**Issue by wdammak** [webkul/hotelcommerce#81]
A large part of hotels use per person pricing with discounts on children accompanied and a compliment in case of single booking.

**Comment by faizansari-bd** [comment]
I agree that many hotels have this kind of business model. Occupancy wise pricings is a feature that is in our road map along with many others. 
We are working on the system but things will take their time to happen if we want to achieve perfection.
But if you are in an urgent need of this feature then please raise a ticket at  https://webkul.uvdesk.com/en/customer/create-ticket/
